qa/tasks/cephadm.py

```diff
@@ -1133,3 +1135,51 @@ def task(ctx, config):
 
         finally:
             log.info('Teardown begin')
+
+
+def add_mirror(s, mirror: str):
```

Comment:
Maybe the proper name should be like `add_containers_registry`?

Reply:
what about `registries_add_mirror_to_docker_io` ?

Reply:
I don't see any `docker.io` word in the package which provides this config file, so maybe it is not good suffix. Package is `libcontainers`, the file path is `/etc/containers/registries.conf`, and as we figured out in other comment it is not _added_ it is more like _set_. So maybe `_set_containers_registry_mirror` and if mirrors accept a list, why not use plural and support lists?